guides/v2.2/cloud/project/project-conf-files_magento-app.md

```diff
@@ -274,7 +274,7 @@ A cron job is well suited for the following tasks:
 -  Or it is long, but can be easily divided into many small queued tasks.
 -  A delay between when a task is registered and when it actually happens is acceptable.
 
-A sample Magento cron job follows:
+By default, every Cloud project has the following default crons configuration to run the default Magento cron jobs:
```

Comment:
so at some point, we'll use the new naming convention -- which is Magento Commerce on the cloud platform,"  Will require variable update, I think.

Reply:
Yes, at one point, we had to make sure the "Cloud" was not used alone. I haven't heard any major directive to change this, but you know it is coming. It is best to use a variable where you can. If you can't, it's not a big deal, but it will help later.